src/containers/ReportePage.js

```diff
@@ -1,30 +1,79 @@
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import ReportList from '../components/reports/ReportList';
+import * as actionsReport from '../actions/reportActions';
 import '../styles/report-page.scss';
 
 class ReportePage extends Component {
   constructor(props, context) {
     super(props, context);
+
+    this.state = {
+        userId: '',
```

Comment:
Porque necesitas mantener el `userId` en el state cuando siempre lo tenes accesible en `this.props.user.id` ???